examples/Atmos/heldsuarez.jl

```diff
@@ -0,0 +1,276 @@
+# # Dry atmosphere GCM with Held-Suarez forcing
+#
+# The Held-Suarez setup (Held and Suarez, 1994) is a textbook example for a simplified atmospheric global circulation model configuration
+# which has been used as a benchmark experiment for development of the dynamical cores (i.e., GCMs without continents,
+# moisture or parametrization schemes of the physics) for atmospheric models.
+# It is forced by a thermal relaxation to a reference state and damped by linear (Rayleigh) friction. This example demonstrates how
+#
+#   * to set up a CLIMA-Atmos GCM configuration;
+#   * to select and save GCM diagnostics output.
+#
+# To begin, we load CLIMA and a few miscellaneous useful Julia packages.
+using Distributions
+using Random
+using StaticArrays
+
+# CLIMA specific imports needed to make this example work (e.g., we will need
```

Comment:
Since new users are likely to read this, we may want to be careful with using `import`, since this is subtly different from `using`

Reply:
Point to Julia base documentation on how `import` and `using` behave (?)

Reply:
Yes, or say `using` when using and `import` when importing